backend/models.py

```diff
@@ -485,6 +485,9 @@ def reject_annotator(self, user, finished_time=timezone.now()):
             annotator_project.status = AnnotatorProject.COMPLETED
             annotator_project.rejected = True
             annotator_project.save()
+
+            Annotation.clear_all_pending_user_annotations(user)
```

Comment:
This is the same behaviour as in `remove_annotator`, and will set the pending annotations to `ABORTED` - is this correct or should we mark them as `REJECTED` instead?  Does that mess things up if we set annotations to rejected in a project that is not configured to permit rejection?

Reply:
I think it makes more sense to set pending annotations as `ABORTED`, as making them `REJECTED` will mean that the annotator will not be given an annotation task for this document again even if they're re-added to the project.